src/syscheckd/db/src/registry.cpp

```diff
@@ -5,12 +5,26 @@
  *
  * @copyright Copyright (C) 2015-2021 Wazuh, Inc.
  */
+
+#include <iostream>
+#include <string.h>
+
+#ifdef WAZUH_UNIT_TESTING
+    #include "fimDBHelpersUTInterface.hpp"
```

Comment:
I propose a tempered factory pattern to avoid this macro that refers to testing in product code.

Reply:
This issue will be addressed in the following [issue](https://github.com/wazuh/wazuh/issues/11332).